frontend/src/app/commonComponents/Card/Card.tsx

```diff
@@ -22,7 +24,11 @@ export const Card: React.FC<CardProps> = ({
           <div className="border-bottom border-base-lighter margin-x-neg-3 margin-top-3"></div>
         </header>
       )}
-      {bodyKicker && (
+      {bodyKicker && bodyKickerCentered ? (
+        <div className="display-flex flex-column flex-align-center">
+          <p className="font-ui-sm text-bold margin-top-3">{bodyKicker}</p>
+        </div>
+      ) : (
         <p className="font-ui-sm text-bold margin-top-3">{bodyKicker}</p>
```

Comment:
This still needs the `bodyKicker` prop wrapped around this. Can you re-add? It's causing an unintended spacing shift.

Reply:
Fixed :thumbsup: